libs/blocks/brand-concierge/brand-concierge.css

```diff
@@ -209,10 +209,18 @@
   line-height: 1.5;
   outline: none;
   width: 100%;
+  resize: none;
+  overflow-wrap: break-word;
+  word-wrap: break-word;
```

Comment:
Aren't these sort of "synonyms"? One or the other should do the trick, did you notice situations where both have an effect?